app/src/main/java/mozilla/lockbox/presenter/AutofillRoutePresenter.kt

```diff
@@ -111,12 +115,18 @@ class AutofillRoutePresenter(
                     "This is a developer bug, fixable by adding an action to graph_autofill.xml"
             )
         } else {
-            val clearBackStack = src.getAction(transition)?.navOptions?.shouldLaunchSingleTop() ?: false
+            val action = src.getAction(transition)
+            val clearBackStack = action?.navOptions?.shouldLaunchSingleTop() ?: false
             if (clearBackStack) {
                 while (navController.popBackStack()) {
                     // NOP
                 }
             }
+
+            action?.let {
+                navController.navigate(it.destinationId, args, it.navOptions)
+                return
+            }
```

Comment:
The sooner we rationalize `AutofillRoutePresenter` and `RoutePresenter` the better.

Reply:
https://github.com/mozilla-lockbox/lockbox-android/issues/508